privacy/scrubber.go

```diff
@@ -199,14 +203,12 @@ func (scrubber) ScrubDevice(device *openrtb2.Device, id ScrubStrategyDeviceID, i
 
 	switch ipv4 {
 	case ScrubStrategyIPV4Lowest8:
```

Comment:
Why do we still need `ScrubStrategyIPV4Lowest8`?

Reply:
We need this to indicate we want to scrun IPv4. There are just 2 options:
```go
const (
	// ScrubStrategyIPV4None does not remove any part of an IPV4 address.
	ScrubStrategyIPV4None ScrubStrategyIPV4 = iota

	// ScrubStrategyIPV4Lowest8 zeroes out the last 8 bits of an IPV4 address.
	ScrubStrategyIPV4Lowest8
)
```
I think we should rename it to something like `ScrubStrategyIPV4Default` or better.

Reply:
Ah ok. Yes, renaming would be good. Perhaps `ScrubStrategyIPV4Subnet` to match IPv6 naming?

Reply:
Right!